options/locale/locale_en-US.ini

```diff
@@ -842,7 +842,7 @@ issues.review.self.rejection = You cannot request changes on your own pull reque
 issues.review.approve = "approved these changes %s"
 issues.review.comment = "reviewed %s"
 issues.review.content.empty = You need to leave a comment indicating the requested change(s).
-issues.review.reject = "rejected these changes %s"
+issues.review.reject = "requested these changes %s"
```

Comment:
this is actually wrong and should be `requested changes %s`

Reply:
Damn.

Can we change it in crowdin? Or do we need another PR?

Reply:
need new PR

Reply:
Just opened PR #5969